docs/dapp/opl/router-protocol/README.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Cross-chain dapps with Router Protocol
+---
+
+# Router Protocol
+
+Router Protocol offers two frameworks for cross-chain interactions:
+- **Router CrossTalk**: Enables stateless and stateful cross-chain messaging
+- **Router Nitro**: Facilitates native cross-chain asset transfers
+
+For guidance on choosing the appropriate framework, refer to Router's [guide].
+
+This documentation focuses on **Router CrossTalk**. If you're primarily
+interested in asset transfers, please consult the [Router Nitro documentation].
+
+[Router Nitro documentation]: https://docs.routerprotocol.com/develop/category/asset-transfer-via-nitro
+[guide]: https://docs.routerprotocol.com/overview/choosing-the-right-framework
+
+## Router CrossTalk
+
+Router CrossTalk is designed to enable cross-chain interactions, allowing
+developers to create decentralized applications (dApps) that operate across
+multiple blockchain networks. This framework supports both stateless and
+stateful operations, providing flexible and efficient communication between
+contracts on different chains.
+
+### Architecture
+
+![Router Architecture](../../images/opl/router-architecture.png)
+*High-level architecture diagram for Router CrossTalk[^1]*
+
+[^1]: The CrossTalk high-level architecture diagram is courtesy of [Router documentation][router-architecture].
+
+[router-architecture]: https://docs.routerprotocol.com/develop/message-transfer-via-crosstalk/key-concepts/high-level-architecture
+
+The **CrossTalk** infrastructure consists of three main components:
+- **Gateway** contracts on source and destination chains
+- **Orchestrators** on the Router chain
+- **Relayers** that forward messages to the Router Gateway contracts
+
+The process flow is as follows:
+1. The dApp contract calls the iSend function on the source chain's Gateway contract.
```

Comment:
As I mentioned, please wrap lines to 80-characters except for code snippets and the second part of the reference-style links. Also md linter suggests new line before start of the list.

```suggestion
The process flow is as follows:

1. The dApp contract calls the iSend function on the source chain's Gateway
   contract.
```